### ISSUE TITLE
Map to base_link?

### ISSUE THREAD

**Issue by abylikhsanov** [mit-racecar/particle_filter#5]
The code in python gives a TF from Map to Base_link. Shouldn't it be map to odom?

**Comment by mboulet** [comment]
Perhaps setting the frame_id values via node parameters would provide flexibility to work with any TF tree design.

**Comment by abylikhsanov** [comment]
Nope, there is a difference between base_link->map and map->odom TF. You can't simply change the frame_id to odom

**Comment by abhilash-nandkumar** [comment]
@abylikhsanov were you able to work around this issue? I have a similar problem

**Comment by matkok** [comment]
it would be great to have this package use same standard as amcl (navigation), such as map->odom tranformation

**Comment by simonrozsival** [comment]
I ran into this problem as well. In the end I commented out https://github.com/mit-racecar/particle_filter/blob/master/src/particle_filter.py#L173 and implemented the `map -> odom` transform myself based on the data it publishes in `/pf/pose/odom` like this: https://github.com/simonrozsival/racer/blob/master/racer-jetson/catkin_ws/src/particle_filter_tf_correction/nodes/fix_tf.py. It's far from elegant, but it works for me.
